### DISCUSSION TITLE
Will you be adding widgets from TailNext?

### DISCUSSION THREAD

**Discussion by glenlong** [arthelokyo/astrowind#80] [General]
I noticed you have a parallel project TailNext, where the design is very similar but not identical.

It has components, such as Testimonials, that AstroWind does not have.

Do you plan to add those components to TailWind?

**Comment by prototypa** [comment]
@glenlong, the quick answer is yes, we are on it. Very soon we will release new widgets in **AstroWind**.

But first we are creating the technological bases and also evaluating the generic design needs and also that the widgets are few but very flexible and useful to compose complex pages easily. In addition there must be a simple entry point to adjust colors, layouts, margins and fonts and adapt more or less to different designs and needs. We are just working on it. 

That's why we still maintain these templates in version 0.x and we haven't completed some perhaps simpler tasks.

**Reply by glenlong** [reply]
Makes sense. Seems like you're working on a lot at the moment! Is it a bad time right now to clone the repo to build a site? I noticed the folder structure changed recently. Are there more major changes coming?

**Reply by prototypa** [reply]
According to our plans it should take about a month to finish the tasks we are doing.

We still dare to make structural changes in some areas since, for example, we still do not have reusable widgets and the developer who uses the template will necessarily have to overwrite each widget and component. We are in that separation between 'core' and customizable code and also doing (partly at your suggestion), making the widgets configurable by properties.
